examples/nextcloud-integration/revad.toml

```diff
@@ -126,9 +126,11 @@ driver = "nextcloud"
 user_layout = "{{.Username}}"
 
 [http.services.ocmd]
-prefix = "ocm"
+
+[http.services.ocmprovider]
 provider = "Reva-Nextcloud"
 endpoint = "http://localhost"
+webdav_root = "/public.php/webdav"
```

Comment:
This should be `/remote.php/dav/ocm` to enable reva to serve ocm resources using the old method

Reply:
Right! and `/ocm` is actually appended as it's the prefix anyway